backend/handlers/handlers.go

```diff
@@ -1530,7 +1530,11 @@ func (handlers *Handlers) postExportLog(r *http.Request) interface{} {
 }
 
 func (handlers *Handlers) getLightningConfigHandler(_ *http.Request) interface{} {
-	return handlers.backend.Config().LightningConfig()
+	lightningConfig := handlers.backend.Config().LightningConfig()
+	for _, account := range lightningConfig.Accounts {
+		account.Mnemonic = ""
```

Comment:
Why set to empty string? Is there a way to remove the mnemonic field?

Reply:
It's now removed by adding `omitempty` to the json descriptor (thanks @Beerosagos)

Reply:
This still removes the mnemonic from the actual lightning account object in the config, so it will be gone afterwards, and also persisted on the next config save() I think.

To do this properly, you'd have to change the handler that returns the data to the frontend, probably recreating a structure with the relevant data for the frontend without the mnemonic.

Reply:
Dang, @benma is right. I didn't realize there was a pointer to the LightningAccountConfig, I thought it would have been a deep copy. Sorry for not noticing it :pray: